.gitlab/test_cpp.sh

```
@@ -104,8 +104,18 @@ kill -s INT $telePID
 # fi
 
+cuda_nvcc=$(command -v nvcc || echo /usr/local/cuda/bin/nvcc)
+cuda_major=$("$cuda_nvcc" --version 2>/dev/null | grep -oP 'release \K[0-9]+' | head -1 || true)
+if [ -n "$cuda_major" ] && [ "$cuda_major" -lt 13 ]; then
+    export NIXL_CI_ALLOW_NVTX_SKIP=1
+fi
+
 # shellcheck disable=SC2154
-gtest-parallel --workers=1 --serialize_test_cases ./bin/gtest -- --min-tcp-port="$min_gtest_port" --max-tcp-port="$max_gtest_port"
+gtest-parallel --output_dir=/tmp --workers=1 --serialize_test_cases ./bin/gtest -- --min-tcp-port="$min_gtest_port" --max-tcp-port="$max_gtest_port"
+echo NIXL_CI_NON_GPU $NIXL_CI_NON_GPU
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)